gno.land/pkg/gnoland/app_test.go

```diff
@@ -822,16 +830,18 @@ func newGasPriceTestApp(t *testing.T) abci.Application {
 	baseApp.MountStoreWithDB(baseKey, dbadapter.StoreConstructor, cfg.DB)
 
 	// Construct keepers.
-	paramsKpr := params.NewParamsKeeper(mainKey, "")
-	acctKpr := auth.NewAccountKeeper(mainKey, paramsKpr, ProtoGnoAccount)
-	gpKpr := auth.NewGasPriceKeeper(mainKey)
-	bankKpr := bank.NewBankKeeper(acctKpr)
-	vmk := vm.NewVMKeeper(baseKey, mainKey, acctKpr, bankKpr, paramsKpr)
-
+	paramsk := params.NewParamsKeeper(mainKey)
+	acctk := auth.NewAccountKeeper(mainKey, paramsk.ForModule(auth.ModuleName), ProtoGnoAccount)
+	gpk := auth.NewGasPriceKeeper(mainKey)
+	bankk := bank.NewBankKeeper(acctk, paramsk.ForModule(bank.ModuleName))
+	vmk := vm.NewVMKeeper(baseKey, mainKey, acctk, bankk, paramsk)
+	paramsk.Register(auth.ModuleName, acctk)
```

Comment:
It might be a good idea to choose either "auth" or "account," but avoid mixing the two.